### PR TITLE
Adds stereolabs cameras and restructures `lab_assets` into robots and sensors

### PR DESCRIPTION
**Pull request by pascal-roth**
# Description

Add configurations for stereolabs cameras. In addition, restructures the lab_assets to be more structured by introducing a subdirectory with all the robot configs and one for the sensors. 

## Type of change

- Breaking change (fix or feature that would cause existing functionality to not work as expected)
- This change requires a documentation update

## Checklist

- [ ] I have run the [`pre-commit` checks](https://pre-commit.com/) with `./isaaclab.sh --format`
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have updated the changelog and the corresponding version in the extension's `config/extension.toml` file
- [ ] I have added my name to the `CONTRIBUTORS.md` or my name already exists there

### CODE REVIEW

File: source/extensions/omni.isaac.lab_assets/omni/isaac/lab_assets/sensors/stereolabs.py [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+# SPDX-License-Identifier: BSD-3-Clause
+
+"""Configuration for Sterolabs Depth Cameras.
+
+The following configuration parameters are available:
+
+* :obj:`ZED_X_NARROW_RAYCASTER_CFG`: The ZED-X Camera with a narrow field of view as an instance of :class:`RayCasterCameraCfg`.
+* :obj:`ZED_X_MINI_WIDE_RAYCASTER_CFG`: The ZED-X Mini Camera with a wide field of view as an instance of :class:`RayCasterCameraCfg`.
+* :obj:`ZED_X_NARROW_USD_CFG`: The ZED-X Camera with a narrow field of view as an instance of :class:`CameraCfg`.
+* :obj:`ZED_X_MINI_WIDE_USD_CFG`: The ZED-X Mini Camera with a wide field of view as an instance of :class:`CameraCfg`.
```
Comment (jtigue-bdai) [inline source/extensions/omni.isaac.lab_assets/omni/isaac/lab_assets/sensors/stereolabs.py]: which lens the 2mm or the 4mm lens?

File: source/extensions/omni.isaac.lab_assets/omni/isaac/lab_assets/sensors/stereolabs.py [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+
+Reference:
+
+* https://store.stereolabs.com/cdn/shop/files/ZED_X_and_ZED_X_Mini_Datasheet.pdf?v=10269370652353745680
```
Comment (jtigue-bdai) [inline source/extensions/omni.isaac.lab_assets/omni/isaac/lab_assets/sensors/stereolabs.py]: This link does not work for me. Perhaps:
https://www.stereolabs.com/store/products/zed-x-stereo-camera -> Resources -> DataSheet
https://www.stereolabs.com/store/products/zed-x-mini-stereo-camera -> Resources -> DataSheet

**Review by jtigue-bdai — APPROVED** [review]
Other that the source of the data and the distinguishing between lenses I think it looks good.

**Comment by pascal-roth** [comment]
There is no need for a review now; will be addressed in November when IsaacSim will also be renamed. Keep the PR open as information.

**Comment by pascal-roth** [comment]
closed for now, will be picked up in the new release

**Pull request closed without merging**